modules/apps/portal/portal-user-locale-options-web/src/main/resources/META-INF/resources/dynamic_include/user_locale_options.jsp

```diff
@@ -31,7 +45,7 @@ Locale userLocale = user.getLocale();
 	<div dir="<%= LanguageUtil.get(request, "lang.dir") %>">
 		<clay:link
 			cssClass="d-block"
-			href='<%= themeDisplay.getPathMain() + "/portal/update_language?redirect=" + URLCodec.encodeURL(themeDisplay.getURLCurrent()) + "&groupId=" + themeDisplay.getScopeGroupId() + "&privateLayout=" + layout.isPrivateLayout() + "&layoutId=" + layout.getLayoutId() + "&languageId=" + themeDisplay.getLanguageId() + "&showUserLocaleOptionsMessage=false" %>'
+			href='<%= themeDisplay.getPathMain() + "/portal/update_language?redirect=" + URLCodec.encodeURL(themeDisplay.getURLCurrent()) + "&groupId=" + groupId + "&privateLayout=" + privateLayout + "&layoutId=" + layoutId + "&languageId=" + themeDisplay.getLanguageId() + "&showUserLocaleOptionsMessage=false" %>'
```

Comment:
can we sort the parameters to be groupId / layoutId / languageId / persistState / privateLayout / ... ?